pkg/local_object_storage/writecache/flush.go

```diff
@@ -112,6 +108,10 @@ func (c *cache) flushDB() {
 		})
 
 		for i := range m {
```

Comment:
Notice also that down below you've got a `len(m) == 0` check, but you've got a different `m` now in some situations, you can ignore some items in this loop, but they will be accounted for later.

Reply:
Fixed!